baseplate/sidecars/event_publisher.py

```diff
@@ -116,7 +116,9 @@ def serialize(self) -> SerializedBatch:
 
 publishCountTotal = Counter("event_publish_total", "total count of published events")
 publishLatency = Histogram("event_publish_latency", "latency for publishing a batch of events")
-publishErrors = Counter("event_publish_errors", "total count of published events", ["error_type"])
+publishErrors = Counter(
```

Comment:
Sorry for commenting on this now instead of a few weeks ago, but how are these metrics different than those of the client that the sidecar uses? Not against it...however this sidecar doesn't use the requests lib from baseplate, so effectively those calls would be uninstrumented (they should be though)

Reply:
Oh cool, yeah switching to the baseplate request lib makes sense to me 👍